### PR TITLE
Instances

### PR DESCRIPTION
**Pull request by paxperscientiam**
Sorry about that. This looks good now.

### CODE REVIEW

File: src/imagelightbox.js [hunk 1]
```diff
@@ -550,6 +554,9 @@
         $(window).on('resize.ilb7', _setImage);
 
         $(document).ready(function() {
+            $(this).on("start.ilb7", function (e,b) {
+                console.log(b);
```
Comment (rejas) [inline src/imagelightbox.js]: so what should happen here besides logging?

File: src/imagelightbox.js [hunk 2]
```diff
@@ -509,13 +511,15 @@
                 inProgress = false;
                 _onStart();
                 $('body').append($wrapper);
-                $wrapper.trigger("start.ilb2");
+                $wrapper
+                    .trigger("start.ilb7",[instances])
```
Comment (rejas) [inline src/imagelightbox.js]: is the event data (aka instances) it yet used somewhere?

File: src/imagelightbox.js (hunk 1 above)
Comment (paxperscientiam) [inline src/imagelightbox.js]: Nothing at the moment; just proof of concept. This is needed for implementation of history API. The `console.log(b)` can be removed.

**Comment by paxperscientiam** [comment]
Would you rather I roll this into a "history" PR?

**Comment by rejas** [comment]
yeah, that sounds like a good idea.

**Pull request closed without merging**